### PR TITLE
Release 0.1.0 with normal extension upgrade support

### PR DESCRIPTION
**Pull request by RobAtticus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+support upgrading between extension versions using the typical
+`ALTER EXTENSION` command, unless otherwise noted in release notes. This
+important step should make it easier to test TimescaleDB and be able
+to get the latest benefits from new versions of TimescaleDB. If you
+were previously using a version with the `-beta` tag, you will need
+to `DROP` any databases currently using TimescaleDB and re-create them
+in order to upgrade to this new version.
+
+**High-level changes**
+* More refactoring to stablize and cleanup code base for supporting upgrades (see above note)
```
Comment (erimatnor) [inline CHANGELOG.md]: Stabilize

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
 # TimescaleDB Changelog
 
+## 0.1.0 (2017-06-28)
+
+**IMPORTANT NOTE**
+Starting with this release, TimescaleDB will now
```
Comment (erimatnor) [inline CHANGELOG.md]: the release notes?

File: CHANGELOG.md (hunk 1 above)
Comment (erimatnor) [inline CHANGELOG.md]: the code base?

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,7 +47,7 @@ the next release.
 * `COPY` and `INSERT` commands now return the correct number of rows
 * Default indexes no longer duplicate existing indexes
 * Cleanup of the Docker image and build processBatches
```
Comment (erimatnor) [inline CHANGELOG.md]: Typo? "processBatches"

Comment (erimatnor) [inline CHANGELOG.md]: Guess this wasn't part of this PR, still...

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+## 0.1.0 (2017-06-28)
+
+**IMPORTANT NOTE**
+Starting with this release, TimescaleDB will now
+support upgrading between extension versions using the typical
+`ALTER EXTENSION` command, unless otherwise noted in release notes. This
+important step should make it easier to test TimescaleDB and be able
+to get the latest benefits from new versions of TimescaleDB. If you
+were previously using a version with the `-beta` tag, you will need
+to `DROP` any databases currently using TimescaleDB and re-create them
```
Comment (erimatnor) [inline CHANGELOG.md]: Make note about backup, or process to migrate data?

File: CHANGELOG.md (hunk 2 above)
Comment (RobAtticus) [inline CHANGELOG.md]: Hmm? It's the release that provides the functionality, not the release notes.

Comment (RobAtticus) [inline CHANGELOG.md]: Ah @cevian pointed out you may have meant the line after. Tweaked.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+**IMPORTANT NOTE**
+Starting with this release, TimescaleDB will now
+support upgrading between extension versions using the typical
+`ALTER EXTENSION` command, unless otherwise noted in future release notes. This
+important step should make it easier to test TimescaleDB and be able
+to get the latest benefits from new versions of TimescaleDB. If you
+were previously using a version with the `-beta` tag, you will need
+to `DROP` any databases currently using TimescaleDB and re-create them
+in order to upgrade to this new version. To backup and migrate data,
+use `pg_dump` to save the table schemas and `COPY` to write hypertable
```
Comment (cevian) [inline CHANGELOG.md]: Should we link to http://docs.timescale.com/getting-started/setup/migrate-from-postgresql#different-db

**Pull request merged**